CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.1.2] - 2022-12-13
+
+### Fixed
+
+- Update `stomp.py` from `7.0.0` to `~=8.1.0`.
```

Comment:
Puts that you changed some log messages too. There may be monitoring on top of messages that may stop working.

Reply:
Nice! I'll add!

Reply:
Added!